app/src/containers/MenuDrawer/listItems.js

```diff
@@ -61,27 +46,44 @@ export const secondaryListItems = requestAmount => (
       <ListItemText primary="Find a partner" />
     </ListItem>
 
-    <ListItem button component={Link} to={"/match-requests"}>
```

Comment:
@juusohuhtivuo I would like to see a visual representation of this list view change. 
Can you upload a screenshot if you don't mind.

Reply:
Sure! Here

![UI_menu](https://user-images.githubusercontent.com/70169366/96449651-d9102e80-121d-11eb-9262-5d4d48ad0121.png)

Reply:
The order of the buttons was changed according to the plan, but I didn't (at least yet) remove the lines separating buttons.

Reply:
line separation is fine I think...
It is separating with a context it seems...
"Personal Stuffs with messaging"
------------------------------------------
"Partnering options"
------------------------------------------
"Redirections and logout"

Reply:
We also considered removing the first line completely but then decided not to do that without proper discussion, and the separation seemed to work logically that way, just like you mentioned.